helm_deploy/values-preprod.yaml

```diff
@@ -62,6 +62,7 @@ env:
   CURIOUS_URL: https://testservices.sequation.net/sequation-virtual-campus2-api/
   DISABLE_REQUEST_LIMITER: true
   PVB_URL: https://manage-a-prison-visit-preprod.hmpps.service.justice.gov.uk/
+  CALCULATE_RELEASE_DATES_URL: https://calculate-release-dates-preprod.hmpps.service.justice.gov.uk/
```

Comment:
this project doesn't look like it has been migrated to use the generic service chart yet so you'll need to add this env var to ./templates/_envs.tpl as well

Reply:
Done thanks.